api/views.py

```diff
@@ -35,9 +37,13 @@ def get_queryset(self):
 class OrganizationViewSet(FilterByUrlKwargsMixin, views.ModelViewSet):
     resource_name = 'organizations'
     lookup_field = 'uuid'
+    # TODO - maybe we should restrict more, but I think it's fine to show
+    # all orgs to everyone through the API
     queryset = Organization.objects.all()
     serializer_class = OrganizationSerializer
     filter_fields = [('study', 'study'), ('user', 'user'), ]
+    http_method_names = [u'get', u'head', u'options']
```

Comment:
There's no more `u'strings'` in Py3. All strings are unicode by default. So while this won't break anything it also doesn't do anything.